### PR TITLE
add all available endpoints to zos environment

### PR DESCRIPTION
**Pull request by Eslam-Nawara**
### Description

add all available endpoints to zos environment

### Changes

add all available endpoints to zos environment

### Related Issues

- #2353 

### Checklist

- [ ] Tests included
- [ ] Build pass
- [ ] Documentation
- [ ] Code format and docstring

### CODE REVIEW

File: qemu/vm.sh [hunk 1]
```diff
@@ -8,7 +8,7 @@ kernelargs=""
 bridge=zos0
 graphics="-nographic -nodefaults"
 smp=1
```
Comment (iwanbk) [inline qemu/vm.sh]: why changing this file?
looks like need separate PR for this

Comment (Eslam-Nawara) [reply qemu/vm.sh]: this was by mistake, removed it

File: pkg/environment/environment.go [hunk 2]
```diff
@@ -253,7 +269,7 @@ func getEnvironmentFromParams(params kernel.Params) (Environment, error) {
 
 	if activation, ok := params.Get("activation"); ok {
 		if len(activation) > 0 {
-			env.ActivationURL = activation[len(activation)-1]
+			env.ActivationURL = activation
```
Comment (rawdaGastan) [inline pkg/environment/environment.go:272]: I didn't get it

File: pkg/substrate_gateway/substrate_gateway.go [hunk 3]
```diff
@@ -48,16 +48,21 @@ func (g *substrateGateway) CreateTwin(relay string, pk []byte) (uint32, error) {
...
-		Str("activation url", activationURL).
+		Strs("activation url", activationURL).
 		Str("terms and conditions link", termsAndConditionsLink).
 		Str("terms and conditions hash", termsAndConditionsHash).
 		Msg("method called")
 	g.mu.Lock()
 	defer g.mu.Unlock()
-	return g.sub.EnsureAccount(g.identity, activationURL, termsAndConditionsLink, termsAndConditionsHash)
+	for _, url := range activationURL {
+		if info, err = g.sub.EnsureAccount(g.identity, url, termsAndConditionsLink, termsAndConditionsHash); err == nil {
```
Comment (rawdaGastan) [inline pkg/substrate_gateway/substrate_gateway.go]: I think we need to check the error according to this https://github.com/threefoldtech/tfchain/pull/986

**Review by rawdaGastan — CHANGES_REQUESTED** [review]

**Comment by Eslam-Nawara** [comment]
created new node to test supporting multiple activation service urls

image
image

**Review by rawdaGastan — DISMISSED** [review]

**Review by rawdaGastan — APPROVED** [review]

**Pull request merged**